src/engine/source/hlp/src/specificParsers.cpp

```diff
@@ -459,7 +554,7 @@ bool parseURL(const char **it, Parser const &parser, ParseResult &result)
     // TODO curl will parse and copy the URL into an allocated
     // char ptr and we will copy it again into the string for the result
     // Check if there's a way to avoid all the copying here
-    char *str;
+    char* str;
     uc = curl_url_get(url.get(), CURLUPART_URL, &str, 0);
     if (uc)
```

Comment:
```suggestion
    if (CURLUcode::CURLUE_OK != uc)
```

Reply:
In this context, the current form makes the reading clearer.